rdd/RDDcounts.py

```
@@ -277,95 +322,229 @@ def create_RDD_counts_all_levels(self) -> pd.DataFrame:
-            reference type, level, count, and group.
-
-        Raises
-        ------
-        ValueError
-            If the  RDD counts have not been yet created
+            Filtered RDD counts dataframe.
         """
         if self.counts is None:
             raise ValueError(
                 "RDD counts have not been created yet. Call create() first."
             )
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)